cineast-core/src/main/java/org/vitrivr/cineast/core/temporal/sequential/SequentialPath.java

```diff
@@ -0,0 +1,102 @@
+package org.vitrivr.cineast.core.temporal.sequential;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.jetbrains.annotations.NotNull;
+import org.vitrivr.cineast.core.temporal.ScoredSegment;
+
+/**
+ * Class to store a sequential scoring path.
+ *
+ * <p>Contains the score and segments and other relevant information for the path</p>
+ *
+ * @author vGsteiger
+ * @created 07.05.2021
+ */
+public class SequentialPath implements Comparable<SequentialPath> {
+
+  private final String objectId;
+  private double score;
+  private final List<ScoredSegment> segments;
+  private int currentContainerId;
```

Comment:
It's not clear to me what `current` means in this context - is it the `latestContainerId`?

Reply:
Yes, renamed in last commit.